jwplayer-appletv-web-app/js/ConfigLoader.js

```diff
@@ -22,11 +22,12 @@ function ConfigLoader() {
 
   self.defaults = {
     "playlists": [],
-    "featuredPlaylist": "",
+    "featuredPlaylist": {},
     "splashScreen": "",
     "bannerImage": "",
     "backgroundColor": "#000000",
-    "analyticsToken": ""
+    "analyticsToken": "",
```

Comment:
@johnluther The `analyticsToken` configuration property is completely optional, and it might be useful to keep it around for debugging.  That being said, it wouldn't make much of a difference if we took it out.